src/com/sun/ts/tests/jaxrs/common/impl/JaxbKeyValueBean.java

```diff
@@ -50,4 +50,4 @@ public JaxbKeyValueBean set(String key, String value) {
     setValue(value);
     return this;
   }
```

Comment:
The copyright date + change at end of file, is not needed for this file

Reply:
Fixed.